instance/models/mixins/openedx_monitoring.py

```diff
@@ -80,12 +140,25 @@ def disable_monitoring(self):
         for monitor in self.new_relic_availability_monitors.all():
             monitor.delete()
 
+        try:
+            alert_policy = NewRelicAlertPolicy.objects.get(instance=self)
+            for channel in alert_policy.email_notification_channels.filter(shared=False):
```

Comment:
Why are we deleting all `email_notification_channels`? Won't this fail with `ProtectedError` if we try to delete a shared notification channel (like ops@opencraft.com)?

Reply:
The `_add_emails` function above reuses existing notification channels. Shouldn't we try to delete, but skip if the channel is shared?

Reply:
@giovannicimolin, the queryset filters on `shared=False` and that should take care of this.